src/main/java/roomescape/Repository/ReservationRepository.java

```diff
@@ -0,0 +1,69 @@
+package roomescape.Repository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.Domain.Reservation;
+import roomescape.Domain.Time;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class ReservationRepository {
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public List<Reservation> getAllReservation() {
+        List<Reservation> reservationList = jdbcTemplate.query(
+                "SELECT r.id as reservation_id, r.name, r.date, t.id as time_id, t.time as time_value FROM reservation as r inner join time as t on r.time_id = t.id",
```

Comment:
java 15부터는 문자열을 여러줄로 작성할 수 있는 multi-line String이 등장했습니다!
쿼리를 조금 더 읽기 쉽게 작성해볼 수도 있을 것 같아요
```suggestion
    String sql = """
    select 
      r.id as reservation_id, 
      r.name, r.date, 
      t.id as time_id, 
      t.time as time_value 
    from reservation as r 
    inner join time as t on r.time_id = t.id
    """;
```

참고자료: https://www.baeldung.com/java-multiline-string

Reply:
가독성이 훨씬 좋네요
36e596dfcd730bfab5ac7f707e37deb5c4b7bb7e 적용했습니다!